bin/publish.js

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env node
+
+'use strict';
+/* eslint-disable node/no-unsupported-features/es-syntax, no-console, no-process-exit, node/no-extraneous-require, node/no-unpublished-require */
+
+/*
+Usage
+
+publish release|beta|canary
+
+Flags
+
+--distTag=latest|lts|beta|canary|release-<major>-<minor>
+--major
+--minor
+--skipVersion
+--skipPack
+--skipPublish 
+--skipSmokeTest
+*/
+
+const debug = require('debug')('publish-packages');
+const chalk = require('chalk');
+const fs = require('fs');
+const path = require('path');
+const { shellSync } = require('execa');
+const cliArgs = require('command-line-args');
+const readline = require('readline');
+const semver = require('semver');
+const projectRoot = path.resolve(__dirname, '../');
+const packagesDir = path.join(projectRoot, './packages');
+const packages = fs.readdirSync(packagesDir);
+
+function cleanProject() {
+  execWithLog(
+    `cd ${projectRoot} && rm -rf packages/*/dist packages/*/tmp packages/*/node_modules node_modules`
+  );
+  execWithLog(`cd ${projectRoot} && yarn install`);
+}
+
+function execWithLog(command, proxyIO = false) {
```

Comment:
Can you leave a comment explaining what proxyIO is used for

Reply:
done